### PR TITLE
Boosted multiclass forest

### PR DESCRIPTION
**Pull request by hectorgon**
@deerzq @Patrick1860 @yolken @christhetree

### CODE REVIEW

File: training/src/main/scala/com/airbnb/aerosolve/training/BoostedForestTrainer.scala [hunk 1]
```diff
@@ -37,10 +37,15 @@ object BoostedForestTrainer {
...
+                                         multiclass : Boolean)
                                          
-  case class ForestResponse(tree : Int, leaf : Int, weight : Double)
-  case class ForestResult(label : Double, sum : Double, response : Array[ForestResponse])
+  case class ForestResponse(tree : Int, leaf : Int)
+  case class ForestResult(label : Double,
+                          labels : Map[String, Double],
+                          sum : Double,
+                          sumResponses : scala.collection.mutable.HashMap[String,Double],
+                          response : Array[ForestResponse])
```
Comment (deerzq) [inline training/src/main/scala/com/airbnb/aerosolve/training/BoostedForestTrainer.scala]: add some comments here explaining items in ForestResponse and ForestResult?

**Comment by christhetree** [comment]
LGTM (nothing jumped out as being horribly wrong), but I would wait for someone else's seal of approval before merging.

File: training/src/main/scala/com/airbnb/aerosolve/training/BoostedForestTrainer.scala (hunk 1 above)
Comment (hectorgon) [inline training/src/main/scala/com/airbnb/aerosolve/training/BoostedForestTrainer.scala]: Done

**Comment by deerzq** [comment]
what's the failure about?

**Pull request merged**